dev/integration_tests/ui/ios/Podfile

```diff
@@ -0,0 +1,38 @@
+# Uncomment this line to define a global platform for your project
```

Comment:
I think _generally_ we don't check in the default generated `ios/Podfile`.  Would you mind removing this file from your patch? The `xcconfig` changes are fine though.